src/index.scss

```diff
@@ -3,3 +3,6 @@
 
 $fa-font-path: "~font-awesome/fonts";
 @import "~font-awesome/scss/font-awesome";
+
+@import "~@edx/paragon/src/SearchField/SearchField";
```

Comment:
To make Paragon styles work properly, we need to include the `scss` file for any component that has non-native Bootstrap here.